src/evaluate/hub.py

```diff
@@ -0,0 +1,63 @@
+from huggingface_hub.repocard import metadata_update
+
+
+def push_to_hub(
+    to: str,
```

Comment:
I'd be in favour of calling this `repo_id` so we use the same naming across the ecosystem.